stackoverlay.js

```diff
@@ -255,7 +347,24 @@ export class StackOverlay {
      * Shows the window preview in from the side it was triggered on.
      */
     showPreview() {
-        let [x, y, mask] = global.get_pointer();
+        // only show if have valid scale
+        const scale = Settings.prefs.edge_preview_scale;
+        if (scale <= 0) {
+            return;
+        }
+
+        // if timeout is enabled, only show if valid timeout (e.g. if SHOW_DELAY is
+        /**
+         * if timeout is enabled, only show if valid timeout (e.g. if SHOW_DELAY is lte
+         * than timeout, then won't see the preview anyway).
+         */
```

Comment:
```suggestion
        /**
         * if timeout is enabled, only show if valid timeout (e.g. if SHOW_DELAY <= timeout,
         * then won't see the preview anyway).
         */
```

quick comment cleanup